tests/TestGranger.py

```diff
@@ -66,10 +66,8 @@ def test_granger_lasso(self):
         dataset = self.syn_data_3d()
         G = Granger()
         coeff = G.granger_lasso(data=dataset)
-        print('Estimated matrix is \n {}'.format(coeff))
+        coeff_truth = np.array([[0.09, 0.1101, 0.1527, 0.1127, 0.0226, 0.1538],
```

Comment:
and what about this?

Reply:
Same, computed from the current version [1ebf232](https://github.com/cmu-phil/causal-learn/commit/1ebf232f0244a60fa7908bdea2b1bdf451da4074).

Reply:
same, please add some comments :)

A good mental practice: write code in a way that, if you check the data a year later, do you know what the code means and how the values are generated?